spring-context/src/main/java/org/springframework/validation/ValidationUtils.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2002-2018 the original author or authors.
+ * Copyright 2002-2021 the original author or authors.
```

Comment:
Nitpicking, this can be restored.

Reply:
Oh man - missed that. Let me restore that.

Reply:
Done